src/libraries/System.Private.CoreLib/src/System/Environment.OSVersion.Browser.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class Environment
+    {
+        private static OperatingSystem GetOSVersion()
+        {
+            return new OperatingSystem(PlatformID.Other, new Version(1, 0, 0, 0));
```

Comment:
Can you add a test for this?

Reply:
There are tests already which are being fixed/enabled as part of #38996